app/graphql/mutations/create_project.rb

```diff
@@ -0,0 +1,22 @@
+module Mutations
+  class CreateProject < BaseMutation
+    argument :name, String, required: true
+    argument :description, String, required: false
+    argument :user_ids, [ID], required: false
+
+    type Types::ProjectType
+
+    def resolve(**_options)
+      result = ::CreateProject.call(
+        project_params: _options ,
```

Comment:
We use underscore in the begging only for unused parameters, you should name it just `options`